plugins/CoreVisualizations/templates/_dataTableViz_tagCloud.twig

```diff
@@ -1,7 +1,7 @@
 {% set cloudColumn = properties.columns_to_display[1] %}
 <div class="tagCloud">
 {% for word,value in cloudValues %}
-    <span title="{{ value.word }} ({{ value.value }} {{ properties.translations[cloudColumn]|default(cloudColumn) }})" class="word size{{ value.size }}
+    <span title="{{ value.word|rawSafeDecoded }} ({{ value.value }} {{ properties.translations[cloudColumn]|default(cloudColumn) }})" class="word size{{ value.size }}
```

Comment:
Do we need to call `$word = Common::unsanitizeInputValue($word);` as well? So far it seems only `wordTruncated` is handled or was `$word` already working?